DuckDuckGo/UserText.swift

```diff
@@ -528,7 +528,13 @@ public struct UserText {
 
     public static let autofillLoginPromptAuthenticationCancelButton = NSLocalizedString("autofill.logins.prompt.auth.cancel", value:"Cancel", comment: "Cancel button for auth during login prompt")
     public static let autofillLoginPromptAuthenticationReason = NSLocalizedString("autofill.logins.prompt.auth.reason", value:"Unlock To Use Saved Login", comment: "Reason for auth during login prompt")
-    public static let autofillLoginPromptTitle = NSLocalizedString("autofill.logins.prompt.title", value:"Use Saved Login?", comment: "Title for autofill login prompt")
+    public static let autofillLoginPromptTitle = NSLocalizedString("autofill.logins.prompt.title", value:"Use a Saved Login?", comment: "Title for autofill login prompt")
+    public static let autofillLoginPromptExactMatchTitle = NSLocalizedString("autofill.logins.prompt.exact.match.title", value:"From This Website", comment: "Title for section of autofill logins that are an exact match to the current website")
+    public static func autofillLoginPromptPartialMatchTitle(for type: String) -> String {
+        let message = NSLocalizedString("autofill.logins.prompt.partial.match.title", value: "From %@", comment: "Title for section of autofill logins that are an approximate match to the current website")
+        return message.format(arguments: type)
+    }
+
```

Comment:
This reminds me we ought to get translations pretty soon

Reply:
Already in progress 🙂